Intersect.Client.Core/MonoGame/Input/MonoInput.cs

```diff
@@ -322,6 +333,9 @@ public override void Update(TimeSpan elapsed)
             {
                 mMouseX = (int)(mouseState.X * ((MonoRenderer)Core.Graphics.Renderer).GetMouseOffset().X);
                 mMouseY = (int)(mouseState.Y * ((MonoRenderer)Core.Graphics.Renderer).GetMouseOffset().Y);
+
+                GameInput.Current.CursorMovementDevice = InputDeviceType.Mouse;
```

Comment:
Ostensibly `GameInput.Current` refers to `this`, it should just be setting `CursorMovementDevice`